releases/UWP/net-native2.0/README.md

```diff
@@ -4,8 +4,16 @@ You can see what was included in each .NET Native 2.0 ([Microsoft.NETCore.Univer
 
 When using Visual Studio these packages require Visual Studio 2017 Version 15.4 or later.
 
-### 6.0.1 (October 5, 2017)
+### UWP 6.0.2 (.NET Native 2.0.1) (November 20, 2017)
+
+#### Quality fixes
+- Fixes an issue when building UWP apps that target RS3 and referenced the Microsoft Store Services SDK and  Microsoft Advertising SDK or Store Engagement SDK [514498]
+- Fixes Crashes due to an error in the compiler's garbage collection reporting (affects ARM apps built with .NET Native) [507618]
+- Improves performance for creating new New UWP Projects. [512775]
```

Comment:
new New UWP Projects --> new UWP projects

Reply:
Additional comment: This particular fix was related to VS installation which users wouldn't get from NuGet, so I'm not sure we should advertise it as a UWP 6.0.2 fix.

Reply:
If someone installed UWP 6.0.2 on VS 15.4, will they see this fix? If so, I think we should include it. Also, I don't plan to write unique release notes for VS, so I wan to include it.